### ISSUE TITLE
The module `./errors` could not be found

### ISSUE THREAD

**Issue by janhesters** [ericelliott/autodux#48]
Using Autodux with a React Native app created by [Expo](https://expo.io/) and Yarn throws:

```bash
Unable to resolve module `./errors` from `/Users/jan/Startup/react-native/os-gastro-admin/node_modules/autodux/dist/index.js`: The module `./errors` could not be found from `/Users/jan/Startup/react-native/os-gastro-admin/node_modules/autodux/dist/index.js`. Indeed, none of these files exist:
  * `/Users/jan/Startup/react-native/os-gastro-admin/node_modules/autodux/dist/errors(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`
  * `/Users/jan/Startup/react-native/os-gastro-admin/node_modules/autodux/dist/errors/index(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`

ABI32_0_0RCTFatal
__37-[ABI32_0_0RCTCxxBridge handleError:]_block_invoke
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
UIApplicationMain
main
start
```

Somehow it loads from `dist` instead of source. Inspecting `node_modules` shows that `dist` is lacking the `errors.js` file.

**Steps to reproduce:**

1. Create a new app using `expo init`
2. Add autodux using `yarn add autodux`
3. Add `redux` and `react-redux` and connect it to a reducer created with `autodux`

You can fix the issue by copying `errors.js` from `src` to `dist`.

**Comment by janhesters** [comment]
I see that you merged a pull request to fix this bug. I don't know whether you already updated the `npm` / `yarn` package, but if you did, the PR didn't fix the issue. I just reinstalled the package, same error. If you didn't update it on `npm`, please disregard this comment 😇

**Comment by grushetsky** [comment]
@janhesters, right, the new version has not been released to npm yet.

**Comment by twhitacre** [comment]
@grushetsky is there anything I can do to help get this released? I can fix it locally, but I have some CI build scripts and they fail, so I've had to revert back to 4.x.x for the time being.

**Comment by ericelliott** [comment]
Published autodux@5.0.1

**Comment by janhesters** [comment]
@ericelliott Thank you!
